public/js/components/tests/Editor.js

```diff
@@ -1,5 +1,7 @@
 "use strict";
 
+if (typeof window == "object") {
```

Comment:
moved the condition up so that the requires are avoided as well.
This was the greg made me do it moment